tests_metricflow/snapshots/test_metric_filter_rendering.py/SqlQueryPlan/BigQuery/test_distinct_values_query_with_metric_filter__plan0_optimized.sql

```diff
@@ -11,7 +11,7 @@ FROM (
   FULL OUTER JOIN (
     -- Aggregate Measures
     -- Compute Metrics via Expressions
-    -- Pass Only Elements: ['listing', 'bookings']
+    -- Pass Only Elements: ['listing', 'listing__bookings']
```

Comment:
Wait, this is weird, the `listing__bookings` alias isn't applied in the rendered SQL internally, so it seems like this comment shouldn't be updated like this.

I don't think it matters but it indicates some potential for edge case mismatches, especially since this is only happening in the optimized plans. Is there some odd interaction with optimizers we need to look out for here?

Reply:
Good question - I had not noticed this so will look into it!

Reply:
THIS WAS JUST DUMB. It's only on the BigQuery and Databricks snapshots, I forgot to rerun them after changing some stuff 🤦‍♀️  adding SQL engine tests to this PR to double check all engines

Reply:
But also prompted me to add a little cleanup - I was building the output column name from the metric spec instead of the group by metric spec. They were the same so it didn't matter, but just in case that changes at some point I updated the logic to pull from the correct spec.